src/ExtLibeventLoop.php

```diff
@@ -10,9 +10,12 @@
 use SplObjectStorage;
 
 /**
+ * Deprecated since v1.2.0, use [`ExtEventLoop`](#exteventloop) instead.
```

Comment:
Should probably use a `@deprecated` annotation similar to https://github.com/reactphp/event-loop/pull/231/files#diff-f1158fbec0bb446d87dbd5346fba683c58ccec1a98b1787511ec7786585e8c31R8 instead :+1:

Reply:
Done 👍